scripts/populate_db.py

```diff
@@ -0,0 +1,169 @@
+    dict['criminal_history_category'] = random.choice(["Category I", "Category II","Category III","Category IV","Category V","Category VI"])
+    
+    return dict 
+
+
+def getSentence(id=None):
+    dict = {}
+    if id is not None:
+        dict['_id'] = id
+          
+    dict['minimum_probation_months'] = random.randint(5,18)
+    dict['fine_dollars'] = random.randint(5,18)
+    dict['minimum_fine_dollars'] = random.randint(500,8000)
+    dict['community_service_hours'] = random.randint(24,180)
+    dict['sentence_type'] = random.choice(["Prison Only","Prison and Alternatives","Probation Only","Probation and Alternatives", "Fine Ordered/No Restitution","Restitution Ordered/No Fine","Both Fine & Restitution Ordered","Prison","Supervision", "Conditional Discharge","Probation","Jail","Conversion","Cook County Boot Camp","Probation Terminated Satisfactorily", "Inpatient Mental Health Services","Probation Terminated Unsatisfactorily", "Conditional Release","Probation Terminated Instanter", "Death"])
+    dict['charge_disposition'] = random.choice(["BFW","Death Suggested-Cause Abated","Finding Guilty","FNG", "FNPC","Nolle Prosecution", "Null","Plea of Guilty", "SOL","Verdict-Not Guilty","Verdict Guilty"])
+    
+    return dict 
+
+
+def getCharge(id=None):
+    dict = {}
+    if id is not None:
+        dict['_id'] = id
+        
+    dict['trial_type'] = random.choice(["Guilty plea","Trial by judge","Trial by jury"])
+    dict['charge_code'] = random.choice(["Antitrust","Armed Robbery","Armed Violence","Arson","Attempt Arson", "Attempt First Degree Murder","Attempt Homicide","Tampering","Tax","Theft by Deception","Theft","Unlawful Restraint"]) 
+    dict['primary'] = random.choice([True, False])  
+    dict['attempted'] = random.choice([True, False]) 
+
+    sentence_list = []
+    for k in range(0, 2):
+        sentence_list.append(getSentence())
+
+    dict['possible_sentences'] = sentence_list
+
+    return dict 
+
+
+def getCase(attorney_id, client_id, id=None):
+    dict = {}
+    if id is not None:
+        dict['_id'] = id
+
+    dict['attorney_id'] = attorney_id
+    dict['client_id'] = client_id    
+
+    charge_list = []
+    for k in range(0, 5):
+        charge_list.append(getCharge())
+
+    dict['possible_charges'] = charge_list
+
+    return dict
+
+
+def main():
+    
+
+    ACCOUNT = os.getenv('AGGREGATOR_DB_ACCOUNT')
+    API_KEY = os.getenv('AGGREGATOR_DB_API_KEY')
+    
+    logging.info('Connecting to Cloudant DB')
+    try:
+        client = Cloudant.iam(ACCOUNT, API_KEY, connect=True) 
+    except Exception as e:
+        logging.error(e)
+        sys.exit(1)
+    
+        
+    logging.info('Creating Client DB')
+    client_db = client.create_database('outcarcerate-client') 
+    client_docs = []
+    for i in range(0, total_clients):
+        client_docs.append(getClient(str(i)))
+
+    logging.info('Populating Client DB')
+    client_db.bulk_docs(client_docs)
```

Comment:
Thank you for making this change. Is this PR ready for review now? @NanoNero

Reply:
Hi, yes. The PR is ready for review now. Thank you.